core/src/HIP/Kokkos_HIP_Instance.hpp

```diff
@@ -55,6 +55,7 @@ namespace Impl {
 
 struct HIPTraits {
   static int constexpr WarpSize       = 64;
+  static int constexpr WarpIndexMask  = 0x003f;
```

Comment:
Comment it is the hexadecimal for 63 for those like me who would need to look it up.